ui/src/types/v2/tasks.ts

```diff
@@ -0,0 +1,9 @@
+import {Task as TaskAPI, Organization, User, Label} from '@influxdata/influx'
+
+export interface Task extends Exclude<TaskAPI, 'labels'> {
```

Comment:
Is there a type from the client lib we could use instead here? Or is this a known will-be-fixed-later-transitional sort of type?

Reply:
This file was supposed to be deleted. rebase conflict